website_hr_department/__openerp__.py

```diff
@@ -26,10 +26,11 @@
     'summary': """
         Display the structure of your departments and their members.
         """,
-    'author': "ACSONE SA/NV,Odoo Community Association (OCA)",
-    'website': "http://acsone.eu",
+    'author': "ACSONE SA/NV,Odoo Community Association (OCA)\
+              Serpent Consulting Services Pvt. Ltd.",
```

Comment:
You have to add a `,` after `(OCA)`, and please join the strings instead of adding all those white spaces to them.